lib/datadog/core/configuration/option.rb

```diff
@@ -290,6 +292,10 @@ def set_value_from_env_or_default
             effective_env = definition.env
             value = coerce_env_variable(ENV[definition.env])
             precedence = Precedence::PROGRAMMATIC
+          elsif definition.backup_env && ENV[definition.backup_env]
+            effective_env = definition.backup_env
+            value = coerce_env_variable(ENV[definition.backup_env])
+            precedence = Precedence::PROGRAMMATIC
           end
```

Comment:
Do you think it makes sense to support a list for the value of `definition.env`, instead of creating `backup_env`?
This way we won't be limited to two environment variables, and it reduces the scope of our configuration API by not adding `backup_env`.

```suggestion
          if (definition_env = definition.env)
            Array(definition_env).each do |env|
              next unless (env_val = ENV[env])
              effective_env = env
              value = coerce_env_variable(env_val)
              precedence = Precedence::PROGRAMMATIC
              break
            end
          end
```